### PR TITLE
add Kamva

### PR DESCRIPTION
**Pull request by hadifarnoud**
Kamva is an ecommerce platform like Shopify. This pull request will detect sites that are built with Kamva

### CODE REVIEW

**Comment by hadifarnoud** [comment]
example site to test this on: http://sanjaaq.com/

**Comment by ScionOfBytes** [comment]
@AliasIO I know I'm turning up on all these issues 😅  but I love the project and wanna see more apps being detected.

If this is good, merge?

File: src/apps.json [hunk 1]
```diff
@@ -3665,6 +3665,15 @@
 			"script": "cf\\.kampyle\\.com/k_button\\.js",
 			"website": "www.kampyle.com"
 		},
+		"Kamva": {
+			"cats": [
+				6
+			],
+			"env": "^Kamva$",
+			"html": "<script[^<]+cdn\\.mykamva\\.ir",
```
Comment (gadcam) [inline src/apps.json]: Why don't you use a script field ? Cf https://github.com/AliasIO/Wappalyzer/wiki/Specification

**Comment by hadifarnoud** [comment]
@gadcam thank you for bringing this to my attention. I added a meta tag as well to detect all sites made by Kamva.

**Comment by hadifarnoud** [comment]
any idea why tests fail @gadcam ?

**Comment by ScionOfBytes** [comment]
@hadifarnoud I think it's because the `generator` property needs to have a string value.

**Comment by hadifarnoud** [comment]
so the docs are out of date? I just followed what was said [here](https://github.com/AliasIO/Wappalyzer/wiki/Specification):
```
"meta":    { "generator": [ "Application Name", "Alternative Application Name" ] },
```

Isn't `"Kamva"` and "Camva" string?

**Comment by ScionOfBytes** [comment]
@hadifarnoud Could be. From what I understand of the rules [here](https://github.com/AliasIO/Wappalyzer/blob/master/schema.json), additional properties of the emta object can only have strings as values. I also checked the [apps.json](https://github.com/AliasIO/Wappalyzer/blob/master/src/apps.json) and couldn't find any `generators` assigned an array value.

**Comment by ScionOfBytes** [comment]
@hadifarnoud I've updated the docs.

**Comment by gadcam** [comment]
```shell
Validating apps.json...
Validation Errors:

Instance is not a required type
uri: urn:uuid:633d6b5f-1dd8-4eb0-824d-630f23818429#/apps/Kamva/meta/generator
schemaUri: urn:uuid:6e98092c-8228-420a-bf5a-e095892ae426#/properties/apps/additionalProperties/properties/meta/additionalProperties

attribute: type
details: ["string"]
npm ERR! Test failed.  See above for more details.
```
@hadifarnoud I don't really know if the problem was formating or the thing you underlined... seems you're right.
In addition if you want to keep both patterns you can write it as it
```js
"meta":{
	"generator": "[CK]amva"
},

**Comment by hadifarnoud** [comment]
@AliasIO any feedback?

**Comment by AliasIO** [comment]
Merged, thanks all.

**Pull request merged**